README.md

```diff
@@ -1,2 +1,180 @@
-# AutoAI-code-generation
-code pattern to show notebook generation feature of AutoAI
+
+<!-- Add a new Title and fill in the blanks -->
+# Generate Python notebook for pipeline models using AutoAI
+
+## Summary
+
+In this code pattern, we will learn how to automatically generate a jupyter notebook model through AutoAI. We will explore, modify and retrain this model pipeline using python code. Lastly, we will deploy this model in Watson Machine Learning using WML APIs.
```

Comment:
capitalize Jupyter.  the jupyter.org folks also capitalize Notebook, but I don't do that.
I'm not sure what a "notebook model" is, but perhaps that is what I'm learning here.